cubedash/_pages.py

```diff
@@ -244,6 +244,38 @@ def about_page():
     return utils.render("about.html")
 
 
+@app.route("/dashboard")
+def dashboard_page():
+    dc = datacube.Datacube(index=_model.STORE.index)
+    dashboard = []
+    import os
+
+    for p, _ in _model.get_products_with_summaries():
```

Comment:
I'm surprised [flake8](https://flake8.pycqa.org/en/latest/) didn't complain about this, but underscore is used for unused variables, and we're using that variable below.

Maybe:
```suggestion
    for product, summary in _model.get_products_with_summaries():
```